setup.py

```diff
@@ -143,7 +143,9 @@ def run(self):
         import click
         import black
 
-        black_opts = []
+        black_opts = [
+            "--extend-exclude", "src/gluonts/nursery/tsbench"
```

Comment:
Let's not do this.

If this code should be part of GluonTS, then it should adhere to its rules.